src/images/money.svg

```diff
@@ -1,6 +1 @@
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////++/P778v55q733pT11Xj00GbzzFnyyVDyyE3///788tT44Z3103HyyE744Z79+Oj446T0z2T99uH33I3++ez224r++ev//vz55q/zy1X77cTzy1f446X//v322oj335b44qH67MH67MD99uPzylTfp0bcoUT103DOiT/AbTrsvUvRj0HEdDvVlULirUf99uLvw0zHez3puEnYnEP9+OfLgj7zzV788dL44Jvls0j44Jr00Wr++vD77sn55Kn324z11Xb11XX00GX55Kj88dH44qD55q3002/33pb224n0z2P++/L778r33pP00GdW9n25AAAAf3RSTlMAAAwxTGV+j5ykphU8Y4oWQGyUEGKJJXYofXMBZKzBz9zq8/n+Tq7D2/BttdXrIYW/4YY9n+mgPk28WbLZxvtB0COjqeImA/Ju+m9Pxav0sYsC5NN8dxQYGUa3uNaW7BfHZo3xDjPCUdFr3+CAgZDSD90/l4yvxK33dCmVDTKeJEz55wAAAAFiS0dEAIgFHUgAAAAHdElNRQfiBgYDMSDCBiN2AAAGjUlEQVRo3r2a+UPURhTHgyLgIngByqFyiYpCUWpF8VakqK2o9aqiIh700J4erccqIDsoVARRkKJWZd3Fhe4Ci8Wr1VZovf6mkjmSTJJJZsPxfspO3vt+djKZN1cEgddGjQ4eExIaNtY2Niw0ZEzw6FHckXwWPi4i0qa2yIhx4UOkP37CxEk2lk2aOGHQnMlR0TYzi46aPAhCzBRTALIpMRYJU2M5CaLFTrVACI8LgCBaXKANE58QIEG0hPiAHpP2TbVNmz4jMSk5JdWempKclDhj+jStS2QAD2ymOjht1uw5drXNmT0rTe04kxeRTsfNnZdhZ1nGvLm0czoXIfMDKihrvt3Y5mdR/gsyzRHZH1KEhXZzW0hRPlpkDAgKylG6L87gIMBHtlgZlmPMWKJwzV1K6Zy/UFZecbHSARyVFyvKyy6cp+4uzVVELjFiKGuxbLmsUHXpcnUNoK2m+pdLVbLL8mV8NVmhcFsph1+pvQr07WrtFdltpSI6mIVYtVryWbNWCq2rdwC2OerrJM+1a2RInj5inZzGc/NJ3LWGGmBsNQ3XiXO+3CjR63QZH8tNUYCDbjQ2AXNraryB/QvkRlmvh5ATyDKpIao5CKJVS80iQzZoEVPlB0VqcbOZEwFA801Sk42SjiZBxkuZ9pNPsfuv3ATRaglkExEq3KxiyOPFFuTbUh4QAoDyFhS4RVJKoBHh0o2tGHErQAQAtzBkq6RFj4xx6va+HTACgNtVqnaP023wXJxA7lhAAHAHpxWpmyib/TNSiNPgb5YQANzFCZLIxcqIGCmZI5979y0y7t9DAlKql1OKNFVD48UNg67X6nQ6W9m3H6Aen0EEtxHEdlKShf5Fo8E/dbndbpfB/TIkIY2M2zEjihSggfVa0yAYTdegRhKRjMKMQroaDUyBNudDzwDD097BhjTQFSlEiB2EiWYgv7OSubfdLZmPSalB48l8IroTMnbhX5+jf1DPCO70uJXWxYLUI5ndWHUXZKzHv/agfO7gQrjd7QyGoxvq7KHGkSL8a69hD/e71eZkeKLevherFomIfSSNoBlIj35gqwbh9nj1XXtQ2iIJZd8AYxG+3g/vPGL8OZeWwazII6i0H+uK08ZifJ0I7/zBiCO6ro5eJ2kZVrM/hkqJWLd4gHEAX5fAO08YLU7eWfH5eMW28TjbWG/WE6hUgnUPCMJBfJkGbzxlhPWSamCiz+kFbHsKtcji5KBwCF8dhuXPTBge9Od7gaE9g1qHsfIh4Qi+KoXlz1lhpD38ncDcnkOtUqx8RPgCX31p2MkV71VXmynjT6j1FVaeKHxNDR1/scKU/aPdjPKCGkSOCsfwVTIsf8mM87n5KS+hVjJWPiaE4asUWF7JjFPlK8P3qhJqpWDlMIGsB1JhucEyoIOG+A0gDqiVipVXCyTPo4xs9AQ6qcfl9hk8LyRGpPnrMWBOqip+7nrwtgc0L0Vhzk/U7cH7XulQXCwn9Xt1lOofL8wYA5QuCcJyUfcPzn4ODbey1B9ZaUvdzznzlSjtIyOf34TxN9T6RspXfHlXJAyIPqQZrPyozrtc44e3FfcNl/i4nGbtQY8f33KOg/LL5Hf5zN4rNA7my+Ogajx/rB/mdOsYq3+g8fw7xXjONS/x+rQI1twH/EPNS77nnl9pponst6qnhZpf7dDOE18xIjs8nE8KvKLmiT/ozHe7WWmRTrs+5rRBd75L5u277SZd3SlRjKY+uvP2nSTPo/VHncFmUifE+I1GWv31B/86Cpiv1dTrqB/V68GkIVgPXqfXgxGMdW3ZIBh9jHWtsI2U4PX5A2DV+lnrcyGPFA3jPoMgndjg/ZK7FhkG+yUjsu+j2b+qsrJ/9a/h/tWI7MONxH6isJl0dtsmsvU65Puiiv3djcO1vysIG6SbFvapu0mI4T61PI4o99v7uPbb+3j32/XPDa6bnxv8F8C5gZxShu/8QxCCZR/6HOc1g/BaeY5TqohmnuMYnkf1a86j+i9bOY8yPFd787bvXcX7nmbQ3PO+4l3f2zfUXf5ztRE5HwwKUp1zJnEQkgI65xQtc4GNogzDea0wEufOgs75+fETiSUa/ZLEE8fVjtzn5/rfAZwUvwPIP1VgLziVL34HcFLrEsh3ACPyPYNg4buMn6x8MBPY9yU/WyCIlsf9nUyeRYJow/+9D7TxK9JPM/VPp68YP1gA4WQnFGr0CxOyh0qf2JmzOcUhoUXnbOeKQkOKc86e4Y78H4e+5pUx0PZkAAAAAElFTkSuQmCC"/></svg>
```

Comment:
don't know if you and Jeremy figured out what's up with this, but flagging just so you don't lose sight of the unexpected base64-image svg

Reply:
i'm pretty sure the reason it is doing this is there is an overlay on top of the svg to give it that grainy look that wasn't achieved with svg. i encountered the same problem when i built this out originally, so i'd say base64 image links are probably fine